### PR TITLE
docs: backstage plugins section

### PR DESCRIPTION
**Pull request by stefanicjuraj**
Fixes https://github.com/daytonaio/enterprise-docs/issues/117
Fixes https://github.com/daytonaio/enterprise-docs/issues/129

### CODE REVIEW

File: src/content/docs/tools/backstage-plugin.mdx [hunk 1]
```diff
@@ -0,0 +1,285 @@
+---
+title: Daytona Backstage Plugin
+description: Using Daytona Backstage Plugin
+sidebar:
+  label: Backstage Plugin
+---
+
+Daytona Backstage Plugin is a set of integrations that enable integration of Daytona's services with Backstage. These plugins provide authentication, Workspace management, and integration capabilities, enabling Backstage users to authenticate via Daytona Keycloak and interact with Daytona Workspaces directly from the Backstage.
+
+To integrate Daytona plugins into your existing Backstage instance, install the following plugins:
```
Comment (ivan-burazin) [inline src/content/docs/tools/backstage-plugin.mdx:10]: @stefanicjuraj I would remove this, and lead the reader through the set by step guide below

File: src/content/docs/tools/backstage-plugin.mdx [hunk 2]
```diff
@@ -0,0 +1,285 @@
+---
+title: Daytona Backstage Plugin
+description: Using Daytona Backstage Plugin
+sidebar:
+  label: Backstage Plugin
+---
+
+Daytona Backstage Plugin is a set of integrations that enable integration of Daytona's services with Backstage. These plugins provide authentication, Workspace management, and integration capabilities, enabling Backstage users to authenticate via Daytona Keycloak and interact with Daytona Workspaces directly from the Backstage.
```
Comment (ivan-burazin) [inline src/content/docs/tools/backstage-plugin.mdx:8]: @stefanicjuraj Repetitive update to: The Daytona Backstage Plugin integrates Daytona's services into Backstage, offering features like authentication, workspace management, and seamless interaction with Daytona Workspaces via Daytona authentication directly within Backstage.

File: src/content/docs/tools/backstage-plugin.mdx [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+
+To integrate Daytona plugins into your existing Backstage instance, install the following plugins:
+
+1. [@daytonaio/backstage-plugin-auth-backend-module-daytona-provider](https://www.npmjs.com/package/@daytonaio/backstage-plugin-auth-backend-module-daytona-provider)
+2. [@daytonaio/daytona-web](https://www.npmjs.com/package/@daytonaio/daytona-web)
+3. [@daytonaio/backstage-plugin-daytona](https://www.npmjs.com/package/@daytonaio/backstage-plugin-daytona)
+
+## Features
+
```
Comment (ivan-burazin) [inline src/content/docs/tools/backstage-plugin.mdx:17]: Features should be updated. here is more context: https://www.daytona.io/dotfiles/daytona-enterprise-meets-backstage

File: src/content/docs/tools/backstage-plugin.mdx [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+// In packages/app/src/App.tsx
+import { daytonaSignInProvider } from '@daytonaio/daytona-web';
+// Add the Daytona Sign-In Provider to the available sign-in providers
+const app = createApp({
+    {/* other api, bind routes here */}
+    components: {
+        SignInPage: props => <SignInPage {...props} auto providers={['guest',daytonaSignInProvider]} />,
+    },
+});
+```
```
Comment (ivan-burazin) [inline src/content/docs/tools/backstage-plugin.mdx:285]: @stefanicjuraj feel like its missing something at the end

**Review by ivan-burazin — APPROVED** [review]

**Review by ivan-burazin — CHANGES_REQUESTED** [review]
@stefanicjuraj 
We should follow the flow of VS code plugin schema. which does include prerequisites. in this case backstage.

Also we should somewhere perhaps in the begindin a link to backstage itsself.


Also HOW TO INSTALL title and the rest of the major section should follow the other plugins.

Lastly would put backstage as the last item in the menu on the left.Ž
Thanks

**Review by ivan-burazin — APPROVED** [review]

**Review by fabjanvucina — APPROVED** [review]

**Pull request merged**